src/app/BugReporter/BugReportForm.cs

```diff
@@ -77,6 +78,7 @@ public BugReportForm()
 
             // ToDo: Displaying report contents properly requires some more work.
             mainTabs.TabPages.Remove(mainTabs.TabPages["reportContentsTabPage"]);
+            Load += (s, e) => ((Form)s!).FixVisualStyle();
```

Comment:
Is this the fix for this?

![image](https://github.com/user-attachments/assets/c5f36c9d-14fd-4cf3-b5ad-bdc33ddc9008)

Reply:
Yes
The themefix is basically the override of bugs in .net, where other general issues should be added.

(It was on my mental list but I had forgotten to add it in the description)